src/scripts/charts/BaseChart.js

```diff
@@ -76,7 +80,8 @@ export default class BaseChart {
 			title: this.title,
 			data: this.raw_chart_args.data,
 			type: type,
-			height: this.raw_chart_args.height
+			height: this.raw_chart_args.height,
+			colors: this.colors
```

Comment:
On second thought, let's add another constraint dict besides `compatible_types`; called `color_compatible_types`, and only use this.colors if it is satisfied. Will be useful for adding new charts in future.